### PR TITLE
Add support for hyperv stimer direct

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
add support for hyperv stimer direct
**Special notes for your reviewer**:

**Release note**:
```
Added support for hv-stimer-direct 
```
Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

**Comment by vladikr** [comment]
@ksimon1 did we have the support but just didn't expose the config correctly?

File: staging/src/kubevirt.io/client-go/api/v1/schema.go [hunk 1]
```diff
@@ -903,6 +903,13 @@ type Features struct {
 	Pvspinlock *FeatureState `json:"pvspinlock,omitempty"`
 }
 
+//
+// +k8s:openapi-gen=true
+type SyNICTimer struct {
+	Enabled *bool         `json:"enabled,omitempty"`
+	Direct  *FeatureState `json:"direct,omitempty"`
```
Comment (xpivarc) [inline staging/src/kubevirt.io/client-go/api/v1/schema.go:910]: Isn't this breaking change?

**Review by xpivarc — CHANGES_REQUESTED** [review]
/hold

**Comment by vladikr** [comment]
> @ksimon1 did we have the support but just didn't expose the config correctly?

ah, got it. ok.

File: staging/src/kubevirt.io/client-go/api/v1/schema.go (hunk 1 above)
Comment (vladikr) [reply staging/src/kubevirt.io/client-go/api/v1/schema.go:910]: @xpivarc in what sense?

Comment (xpivarc) [reply staging/src/kubevirt.io/client-go/api/v1/schema.go:910]: Sorry, I didn't checked the `FeatureState` struct. All good since we have `Enabled` here.

**Review by xpivarc — APPROVED** [review]
/hold cancel

**Pull request merged**